pom.xml

```diff
@@ -664,7 +671,6 @@ THE SOFTWARE.
                 <bannedDependencies>
                   <excludes>
                     <exclude>org.sonatype.sisu:sisu-guice</exclude>
-                    <exclude>log4j:log4j</exclude>
```

Comment:
I now modified the enforcer-rules to limit the exclusions to scopes compile and runtime.